ci-operator/step-registry/baremetalds/devscripts/setup/baremetalds-devscripts-setup-commands.sh

```diff
@@ -10,9 +10,11 @@ echo "************ baremetalds devscripts setup command ************"
 # shellcheck source=/dev/null
 source "${SHARED_DIR}/packet-conf.sh"
 
+INSTALL_STAGE="initial"
 # Get dev-scripts logs and other configuration
 finished()
 {
+  echo "$? $INSTALL_STAGE" > "${SHARED_DIR}/install-status.txt"
```

Comment:
We do not have a doc for this. The intention is to save the install status in a file so that later on another step (must-gather in this case) will create a junit test case based on the status. We are doing this because, as of now, there is no consistent way to define an install step. Therefore post analysis have to maintain a list of names for identifying the install steps. Whenever a new step is added, our analysis tool could be broken and we need to update the list for it to work again. By adding this generic install step, we can simplify that work and have some consistency. The generated junit should reflect the status of the existing install step.